financial-connections/src/test/java/com/stripe/android/financialconnections/features/success/SuccessViewModelTest.kt

```diff
@@ -74,7 +74,7 @@ internal class SuccessViewModelTest {
             activeAuthSession = ApiKeyFixtures.authorizationSession(),
             activeInstitution = ApiKeyFixtures.institution()
         )
-        whenever(getCachedAccounts()).thenReturn(accounts.data)
```

Comment:
Not necessary in this pull request, but a good illustration of why we want fakes instead of mocks. We should get in the habit of modeling our use cases as interfaces and then injecting a fake in the test with some hard-coded data.